ecs-cli/modules/commands/regcreds/regcreds_command.go

```diff
@@ -57,5 +57,13 @@ func regcredsUpFlags() []cli.Flag {
 			Name:  flags.NoRoleFlag,
 			Usage: "[Optional] If specified, no task execution role will be created.",
 		},
+		cli.BoolFlag{
+			Name:  flags.NoOutputFileFlag,
+			Usage: "[Optional] If specified, no output file for use with 'compose' will be created.",
```

Comment:
Could you remind me if this file is meant to be a place for things other than registry credentials to be written?

Reply:
Right now it's meant to 1) provide output for use with compose and 2) just give persistent record of what creds (& role) were created for which registries. Do you think this text is inaccurate or misleading in some way?